### DISCUSSION TITLE
Tutoring? Anyone interested

### DISCUSSION THREAD

**Discussion by TalonCaptain** [freeplane/freeplane#566] [Help]
I'm an old educational enthusiast and am enthralled with Freeplane, but find it difficult to manage at times. I am particularly interested in convincing my daughter, who is starting a PhD next month, to use it. I would really appreciate it if I could find someone willing to zoom with me for 30 minutes occasionally to save me tons of time and aggravation slogging through tutorials. Apologies if this has been beaten to death but I did try to search, with no success.
Cheers

**Comment by macmarrum** [comment]
Freeplane can indeed be enthralling, and with its many features, it can also be difficult.

It's great that you want to spread the love for FP.

So far the kind of help that's been given in this forum was mainly in the form of posts/discussions. Many such posts point to documentation, others answer questions or give some explaination. So screen-sharing would be something new. 

Is there any particular area in Freeplane of interest to you? Also, how familiar are you with Freeplane (been using it for a week or maybe 5 years)? Your answers might encourage people to offer such on-line sessions.

In the meantime, we've recently had our docs reviewed (at least some of the content). Have you see the new site? E.g. https://docs.freeplane.org/#/user-documentation/formatting-maps-and-nodes

**Reply by TalonCaptain** [reply]
Thank you so much for your considerate reply. 

My primary area of interest is to be prepared to give my daughter an impressive sales pitch by working with her on how she can use the tool for her PhD in Sociology, and I figure I only have one shot at it, else she would roll her eyes at yet another crazy idea from her old man.

I find that I have been able to solve every problem I have encountered, but it requires a lot of effort on my part. An example: for some reason my MM was stuck on "viewing" and I didn't realize it, it took me about 5 minutes to figure out something so simple. My question list has "connector management" in it, I haven't broken the code of how to manipulate the geometry of connectors, I have invested 10 minutes in trying to figure out how to manipulate those 2 handle-like thingies without success. I'm confident that I will be able to figure it out, but if I could pay someone for some coaching, I might avoid some frustration as I expand my capabilities. Also, I hate being a pain in the ass and clogging up this board with stupid questions that have already been answered elsewhere. This board seems tailored for pretty high-end users, I need the shallow end.

I took a quick peek at the link you provided and it looks helpful, I will invest more time checking it out. I have probably only 30 hours of Freeplane usage, but spread out unevenly over 5 years

Many thanks again

**Reply by macmarrum** [reply]
Thanks for your in-depth answer. I completely understand the idea of being prepared to give a sales pitch on how to use Freeplane to someone close.

You have probably already seen the feature rundown at https://docs.freeplane.org/#/

> An example: for some reason my MM was stuck on "viewing" and I didn't realize it, it took me about 5 minutes to figure out something so simple.

This has been seen in the forum as the culprit of some frustration.

> My question list has "connector management" in it, I haven't broken the code of how to manipulate the geometry of connectors,

It's also among questions that people ask in the forum. There's a description of the functionality, if you want to give it a go before a tutor comes forward. _BTW, the page is in the "attic", because some other parts still need completion._
https://docs.freeplane.org/#/attic/new-wiki-initiative/understanding-nodes?id=connectors

**Reply by TalonCaptain** [reply]
Wowzer! I really am impressed with the consideration given rookies, I will check it out, thanks tons.

**Comment by dpolivaev** [comment]
I am not sure if you are looking for paid or unpaid tutoring. Could you please clarify how you mean it?

**Reply by TalonCaptain** [reply]
Thanks for responding Sir, my reply to Macmarrum might give a little more context to my request. It wouldn't be a regular gig, just 30 minutes here or there whenever you have some free time. (My time zone is GMT-7) I would be happy to compensate a coach for their time, although I would have little idea on what might be mutually acceptable. I'm thinking that it would depend on:
Familiarity with FP
Coaching skills
English language skills (unfortunately it's the only language I'm fluent in)
Cheers

**Reply by dpolivaev** [reply]
I have a friend in your time zone having in my opinion good teaching skills. He also created a YouTube channel with Freeplane Tutorials, https://youtube.com/c/FreeplaneEnglish

I want to get in touch with him first and if you both are interested you can try it out.

**Reply by TalonCaptain** [reply]
I have watched the first dozen or so, thank you for that, they look like they will be very helpful going forward. I would appreciate you checking in with him. Once again, though, it wouldn't be a steady gig.

**Comment by euu2021** [comment]
For a research workflow, you will probably want to use connectors a lot, as a complement to the hierarchical structure of the nodes. To improve the use of connectors, I recommend taking a looking at [this script](https://sourceforge.net/p/freeplane/discussion/758437/thread/00a59ea0d6/), and reading [this thread](https://sourceforge.net/p/freeplane/discussion/758437/thread/986addc0ee/).

**Reply by TalonCaptain** [reply]
Thank you so much eu2021, I envision two fundamentally different uses of a MM--Organization of a ...project, say, with lots of different components branching out in two dimensions (I'd love to have it in 3). The other is a linear, primarily one-dimensional step by step process. I'm thinking left-to-right orientation. E.g.: research, thesis, lit review, write, edit, etc. Possibly being recursive. Of course, you would intermix the two, but I'm thinking connectors would be essential to this sequential process.

I look forward to reading the thread (but am afraid of a thing called "script," I will try to muster my courage.

**Reply by SchneiderCoppola** [reply]
If I could be of any help you can e-mail me at:

schneidercoppola@gmail.com

I'm the creator of the channel Freeplane Tutorials:
https://www.youtube.com/c/FreeplaneEnglish

I lived for many years in the US and currently I ran an English Course, so I feel really comfortable communicating in English.

Since I've been living in South America any First World Currency is extremely strong over our South America Currency, so, If you intend to get any kind of zoom call Freeplane live tutoring It wouldn't cost much depending on where you live.

I've been very busy this year, but I'm pretty sure we can try to set something up.

Best Regards.

**Reply by TalonCaptain** [reply]
Thanks, I'll be in touch.

**Reply by dpolivaev** [reply]
Hello @TalonCaptain,
If you want to get ideas and support from the community, you can always ask your questions here.
But if you want to get regular help and insights you can contact @SchneiderCoppola using his email.
Does it help? What is your current preference? I am just curious.
Cheers, Dimitry

**Comment by euu2021** [comment]
> I look forward to reading the thread (but am afraid of a thing called "script," I will try to muster my courage.

It's interesting that you say this, because I was recently thinking how the word "script" is indeed scary for people who are not used to coding. Other words, like "macros" or "automation", aren't very inviting, neither. 

But, using the scripts is easy:

- follow the first steps described in the [Scripting page](https://docs.freeplane.org/#/scripting/Scripting) of the documentation (it's enough to stop at the Hello Controller section)
- insert these 2 scripts in the scripts folder: the ConnectedCluster that I created in the thread; and [this one](https://sourceforge.net/p/freeplane/discussion/758437/thread/7b115122/#60c4) that I linked in the thread
- restart Freeplane
- and it's ready to use by navigating to `Tools->Scripts->...` . I like to set 3 hotkeys, so it's easy to navigate: Script Filter Connected; Script Connected Cluster (to expand); No filtering (to clean).

---
Some other things that can be useful for your use case:

- I can't live without the node minimizer script created by macmarrum, as I describe [here](https://github.com/freeplane/freeplane/discussions/401#discussioncomment-2845218)
- I like [this format](https://github.com/freeplane/freeplane/discussions/455#discussioncomment-2951222) (and a screenshot [here](https://github.com/freeplane/freeplane/discussions/560#discussioncomment-3392973)) to study debates, which are very frequent in Humanities. Some days ago, Dimitry implemented a [new feature](https://github.com/freeplane/freeplane/discussions/560) that will make it much easier to use this format. This new feature has a potential for many other things, but we just started exploring it.
- when a map has multiple sections (research, draft, thesis, lit review...), inserting a cloud to the parent node of each section will change the background color for the whole branch. This color coding makes it easier to manage multiple tabs and quickly switch between them.
- talking about managing multiple tabs, I wrote a bit about it in [this thread](https://sourceforge.net/p/freeplane/discussion/758437/thread/9d5be10a28/). We also recently had [this discussion](https://github.com/freeplane/freeplane/discussions/512) about how to "save" the multiple views in order to quickly reopen them after starting up Freeplane.
- I recently created [this script](https://github.com/freeplane/freeplane/discussions/548). But, it's still not very user-friendly, so I may be too complicated for your situation
- The Jumper add-on is essential for big maps. I suggest [this version](https://github.com/freeplane/freeplane/discussions/374#discussioncomment-2771504), because I fixed a small problem of the official version.
- As you will be trying to convince a person that is probably used to word processors, I think you will find [this thread](https://sourceforge.net/p/freeplane/discussion/758437/thread/b29fb5a56b/) interesting. There, I play with the idea of a "word processor" mode, as a more gentle introduction to Freeplane, in this kind of situation.

---

> The other is a linear, primarily one-dimensional step by step process. I'm thinking left-to-right orientation.

I'm not sure if I understand what you mean by `left-to-right orientation`. Breaking the default FreePlane layout is usually not an easy task. I once experimented with that to create [this timeline format](https://sourceforge.net/p/freeplane/discussion/758437/thread/49de608ff9/), but it's a very limited approach. Please, elaborate a bit more about that point.

**Reply by TalonCaptain** [reply]
Two admin points before I reply to the specifics of your post:
1)	I am a bit overwhelmed by the generosity evidenced by the replies, I won’t waste anyone’s time by mentioning it again, but I will continue to be honored by any and all responses.
2)	I don’t know if it would be better to address your last query in a new/different thread, but I trust someone will advise if that is the case.

I don’t know how to paste sections of a prior post to provide context to my comment so I will just use quotation marks.
“The other is a linear, primarily one-dimensional step by step process. I'm thinking left-to-right orientation.”
“I'm not sure if I understand what you mean by left-to-right orientation. Breaking the default FreePlane layout is usually not an easy task. I once experimented with that to create [this timeline format](https://sourceforge.net/p/freeplane/discussion/758437/thread/49de608ff9/), but it's a very limited approach. Please, elaborate a bit more about that point.”

My most formidable current challenge with FP reflects more of a psychological hang-up than a practical one. I find the elegance of its concept breathtaking and want to work with it rather than against it. Conceptually, I view child nodes as elaborations of parent nodes; subsequent levels provide details of prior, higher levels. When you open google earth the root node is visible, drilling down to different levels provides more details. I see no value in considering “sequence” in the organization of google earth. Lake Erie does not come before, nor after, the United States, fitting my view of a MM.

.
Now for an example of my conundrum: 
The root is “Dinner Party” and one of the numerous descendants is rice. I want to depict the 3 discrete, sequential steps required to make rice, “boil water,” “add rice,” “cook for 20 minutes” as three separate nodes. I read left to right so I want “boil water” to be on the left. My hang-up is that the relationships do not fit with what I consider either parent-child or sibling. Conceptually “boil water” should not be a parent to “add rice.” “Add rice” is not a component of “boil water,” it does not clarify it, providing more details about boiling water. And I think it would be worse to consider "adding rice" to be a parent to "boiling water." In your impressive timeline example you state that “Each year is a child of the prece[eding] year.” My hang-up is that adding rice is not a child. This process of making rice lends itself to a flow chart in my thinking. I have tried to make the 3 steps “free” nodes, relying on connectors to indicate sequential relationships, hence my interest in connectors. But, as ridiculous as it may sound, I don’t want to “[Break] the default FreePlane layout.”

I am intending to address the rest of your considerate response later, I am a bit swamped with all of this right now.
Cheers, Per

**Reply by euu2021** [reply]
> I want to depict the 3 discrete, sequential steps required to make rice, “boil water,” “add rice,” “cook for 20 minutes” as three separate nodes. I read left to right so I want “boil water” to be on the left. My hang-up is that the relationships do not fit with what I consider either parent-child or sibling. Conceptually “boil water” should not be a parent to “add rice.” “Add rice” is not a component of “boil water,” it does not clarify it, providing more details about boiling water. And I think it would be worse to consider "adding rice" to be a parent to "boiling water." In your impressive timeline example you state that “Each year is a child of the prece[eding] year.” My hang-up is that adding rice is not a child.

I think that it's a matter of perspective. A child node can be seemed as a component; but it can also be seemed as a succeeding entity (like real children?). In other words, the passage of time, and a sequence of events, can be understood as a parent-child relationship, because each single moment can be considered as an entity derived from the previous moment, just like the child can be seemed as an entity derived from the parent. So, for example, in my timeline, I chose the "yearly" granularity, but the logic is still sound, according to that perspective. And, even multiple children makes sense here, if we are representing multiple events happening "at the same time" (which is a variable that changes according to the scale chosen).

> I have tried to make the 3 steps “free” nodes, relying on connectors to indicate sequential relationships, hence my interest in connectors. But, as ridiculous as it may sound, I don’t want to “[Break] the default FreePlane layout.

Now, if I understood correctly, your idea is to place free positioned nodes, lined from left to right, in order to represent a sequence of events?
Are you trying to create something similar to this layout:

![image](https://user-images.githubusercontent.com/77707706/185264194-e38b805f-9f43-4c48-a1fc-12f67a634a6f.png)

Did you notice that the branch of free positioned nodes overlap with other nodes when they grow to sides:

![image](https://user-images.githubusercontent.com/77707706/185264306-08b9a20b-1bcd-4fd7-96ef-5d06bc6bcaa3.png)


---

>I don’t know how to paste sections of a prior post to provide context to my comment so I will just use quotation marks.

Link [to the Github documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-text).
